index.md

```diff
@@ -0,0 +1,8 @@
+# PRI 
+The versions are indicated by a version identification of N.0.X. This shall be read as the actually released version of the Artifact in the ARM is the version N.0 with the highest X. Once a release has been made, N is stepped to N+1.
+
+## 0.0.X
```

Comment:
check the version and update it properly here

Reply:
there is no need of this file you can remove it, anyway we are maintaining changelog.md file

Reply:
okie